Jenkinsfile

```diff
@@ -185,6 +225,7 @@ pipeline {
                                            yum --disablerepo=\\* --enablerepo=build\\* makecache"''',
                                 junit_files: null,
                                 failure_artifacts: env.STAGE_NAME
+                        job_status_update()
```

Comment:
Doesn't this need to wrap the above runTest() the way it does in other stages?

Reply:
No, just an example showing two different ways of updating the job status.
For a simple status, just call jobs_status_update() with no parameters.
For a more complex status, job_status_update() must wrap a build step that returns a Map.